sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.Publish.ZipDeploy.targets

```diff
@@ -50,7 +50,8 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
       DeploymentPassword="$(Password)"
       SiteName="$(DeployIisAppPath)"
       PublishUrl="$(PublishUrl)"
-      UserAgentVersion="$(ZipDeployUserAgent)"/>
+      UserAgentVersion="$(ZipDeployUserAgent)"
+      UseBlobContainerDeploy="$(UseBlobContainerDeploy)"/>
```

Comment:
from where will the value of `UseBlobContainerDeploy` be set? Is that something VS will set before evaluating this msbuild target during publish?

Reply:
Yes. VS IDE creates a publish profile that uses this task, and that contains all the properties defined for it.

When VS creates a publish profile for Function App (Flex Consumption), `UseBlobContainerDeploy=True` will be included in the profile.